websockets/py35/py35_test_client_server.py

```diff
@@ -3,9 +3,9 @@
 
```

Comment:
The problem here is that the file contains syntax that Python < 3.5 doesn't understand. Merely importing the file causes trouble, before any code can execute.